make_ghpages/commit.sh

```diff
@@ -1,23 +1,26 @@
-#!/bin/bash
-set -e # Exit with nonzero exit code if anything fails
+#!/usr/bin/env bash
 
-# commit sha of master branch
+# Exit with nonzero exit code if anything fails (-e)
+# Print all executed lines (-x)
+set -ex
+
+# Commit sha of master branch
 SHA=`git rev-parse --verify HEAD`
 TARGET_BRANCH="gh-pages"
 
-if [ "$GITHUB_ACTIONS" != "true" ]; then
+if [ "${GITHUB_ACTIONS}" != "true" ]; then
     echo "Skipping deploy; just doing a build."
     exit 0
 fi
 
-# move html to temp dir
+# Move html to temp dir
 mv make_ghpages/out ../page-build
-git checkout $TARGET_BRANCH || git checkout --orphan $TARGET_BRANCH
+git checkout ${TARGET_BRANCH} || git checkout --orphan ${TARGET_BRANCH}
```

Comment:
Technically here the curly brackets are not needed. However since you are changing this, better to add quotes in case of a space for instance, or some other character, in the branch name
```suggestion
git checkout "${TARGET_BRANCH}" || git checkout --orphan "${TARGET_BRANCH}"
```